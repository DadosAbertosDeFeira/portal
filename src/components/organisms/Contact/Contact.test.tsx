import { render } from "@testing-library/react";
import { BiSearch } from "react-icons/bi";

import type { ContactSocialMedia } from ".";
import { Contact, type ContactProps } from ".";

describe("Contact", () => {
  const socialMediasMock: ContactSocialMedia[] = [
    { icon: BiSearch, name: "Icon", url: "https://www.test.com" },
  ];

  const makeSut = ({
    socialMedias = socialMediasMock,
    ...props
  }: Partial<ContactProps>) => {
    return render(<Contact socialMedias={socialMedias} {...props} />);
  };

  it("should renders correctly", () => {
    const { asFragment } = makeSut({});
    expect(asFragment).toMatchSnapshot();
  });
});
