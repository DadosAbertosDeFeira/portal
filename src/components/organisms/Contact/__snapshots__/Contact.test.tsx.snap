// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should renders correctly 1`] = `[Function]`;
